contracts/strategies/convex/base/ConvexStrategyUL.sol

```diff
@@ -210,14 +210,20 @@ contract ConvexStrategyUL is IStrategy, BaseUpgradeableStrategyUL {
     rewardTokens.push(_token);
   }
 
-  // We assume that all the tradings can be done on Sushiswap
+  /**
+   * liquidates rewards via defined dexes such as uni / sushi
+   * and converts accordingly for reinvesting underlying
+   */ 
   function _liquidateReward() internal {
     if (!sell()) {
-      // Profits can be disabled for possible simplified and rapoolId exit
+      // Profits can be disabled for possible simplified and rapid exit
       emit ProfitsNotCollected(sell(), false);
       return;
     }
 
+    // TODO: check for sell floor is missing?
```

Comment:
I left the sellFloor out when creating this because of the multiple rewardTokens. In the future we could maybe add it as a mapping, setting separate sellFloor per rewardToken. For now it's not very important.

Reply:
I was guessing that, thank you for confirming